api/v1alpha1/contour_types.go

```diff
@@ -89,6 +96,49 @@ type NamespaceSpec struct {
 	RemoveOnDeletion bool `json:"removeOnDeletion,omitempty"`
 }
 
+// EndpointPublishing defines the schema to publish network endpoints and represents
+// the publishing type.
+type EndpointPublishing struct {
+	// Type is the type of publishing strategy to use. Valid values are:
+	//
+	// * LoadBalancerService
```

Comment:
nit: Should we drop `Service` from these names and just call them `LoadBalancer`, or `NodePort` to match Kubernetes?

Reply:
@stevesloka as I mention in https://github.com/projectcontour/contour-operator/pull/151#issuecomment-742161550, future types may not use a k8s `Service` resource. I thought it made sense to use types that are backed by a k8s service to include "Service" in the name. Take `LoadBalancerService` for example, what if a future `LoadBalancer` type is added that creates a hardware load balancer outside the k8s cluster?